docroot/sites/grad.admissions.uiowa.edu/modules/grad_admissions_core/grad_admissions_core.module

```diff
@@ -0,0 +1,6 @@
+<?php
+
+/**
+ * @file
+ * Contains grad_srop.module.
```

Comment:
can this be updated?

Reply:
Whoops. Yes. Sorry about that... was using it while talking through config split scenarios.